### ISSUE TITLE
Hargreaves PET calculation

### ISSUE THREAD

**Issue by zhaod95** [monocongo/climate_indices#376]
Hi James,

I find that PET can be calculated utilizing Thornthwaite and Hargreaves equations described in the documentation. But I can only find the command to calculate Thornthwaite PET:
process_climate_indices --index pet --periodicity monthly --netcdf_temp /data/nclimgrid_lowres_tavg.nc --var_name_temp tavg --output_file_base <out_dir>/nclimgrid_lowres --multiprocessing all_but_one
I have no idea how to calculate daily Hargreaves PET using tmin and tmax data. Your help will be much appreciated.

Thanks!
Dan

**Comment by monocongo** [comment]
Yes, Dan, Hargeaves PET is not available yet. Sorry, I haven't had much time to devote to this project lately and this is one of the issues that's been left undone (see issue #102). There is code for a [Hargreaves PET computation with daily values](https://github.com/monocongo/climate_indices/blob/master/src/climate_indices/eto.py#L304) but I haven't tested/vetted the results. Feel free to use that code and please let me know how it works for you and/or what you think needs to be done to make this more useful (probably an option to use Hargreaves PET in the main processing script's arguments).
